libraries/Microsoft.Bot.Builder/BotFrameworkStreamingExtensionsAdapter.cs

```diff
@@ -0,0 +1,695 @@
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>A task that represents the work queued to execute.</returns>
+        public async Task<HttpOperationResponse> DeleteConversationMemberAsync(string conversationId, string memberId, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (string.IsNullOrWhiteSpace(conversationId))
+            {
+                throw new ArgumentNullException(nameof(conversationId));
+            }
+
+            if (string.IsNullOrWhiteSpace(memberId))
+            {
+                throw new ArgumentNullException(nameof(memberId));
+            }
+
+            var route = string.Format("/v3/conversations/{0}/members/{1}", conversationId, memberId);
+            var request = Request.CreateDelete(route);
+
+            return await SendRequestAsync<HttpOperationResponse>(request, cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Lists the members of the specified activity.
+        /// Throws <see cref="ArgumentNullException"/> if conversationId or activityId is null, empty, or whitespace.
+        /// </summary>
+        /// <param name="conversationId">The id of the conversation the activity is a part of.</param>
+        /// <param name="activityId">The id of the activity to fetch the members of.</param>
+        /// <param name="cancellationToken">Optional cancellation token.</param>
+        /// <returns>List of Members of the given activity.</returns>
+        public async Task<IList<ChannelAccount>> GetActivityMembersAsync(string conversationId, string activityId, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (string.IsNullOrWhiteSpace(conversationId))
+            {
+                throw new ArgumentNullException(nameof(conversationId));
+            }
+
+            if (string.IsNullOrWhiteSpace(activityId))
+            {
+                throw new ArgumentNullException(nameof(activityId));
+            }
+
+            var route = string.Format("/v3/conversations/{0}/activities/{1}/members", conversationId, activityId);
+            var request = Request.CreateGet(route);
+
+            return await SendRequestAsync<IList<ChannelAccount>>(request, cancellationToken).ConfigureAwait(false);
+        }
+
+        private async Task<T> SendRequestAsync<T>(Request request, CancellationToken cancellation = default(CancellationToken))
+        {
+            try
+            {
+                var serverResponse = await _server.SendAsync(request, cancellation).ConfigureAwait(false);
+
+                if (serverResponse.StatusCode == (int)HttpStatusCode.OK)
+                {
+                    return serverResponse.ReadBodyAsJson<T>();
+                }
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex.Message);
+            }
```

Comment:
Consider an overload on LogError that takes an Exception. That way you just say "Log" everywhere and it does the right thing.